src/components/Sidebar/NavContent.jsx

```diff
@@ -95,6 +96,7 @@ function NavContent() {
 
             {address && (
               <div className="wallet-link">
+                <Davatar size={20} address={address} style={{ marginRight: 8 }} generatedAvatarType="jazzicon" />
```

Comment:
blockies need a quick styling adjustment to pick up this `marginRight` 
![image](https://user-images.githubusercontent.com/80423742/147484931-d833e49d-746c-40a6-bb46-76d3ce7cc332.png)